mode: svl
velocity: 8
steer_sensitivity: 0.1
debug_window: True
birdeye_warp_param:
  top: 100
  bottom: 180
  top_width: 120
  bottom_width: 180
  birdeye_width: 180
filter_thr_dict:
  saturation_thr: [120,160]
  x_grad_thr: [0,255]
  red_thr: [60,160]